pandas/io/formats/templates/latex.tpl

```diff
@@ -9,9 +9,12 @@
 {% if position_float is not none%}
 \{{position_float}}
 {% endif %}
-{% if caption %}
+{% if caption and caption is string %}
 \caption{% raw %}{{% endraw %}{{caption}}{% raw %}}{% endraw %}
 
+{% elif caption and caption is sequence %}
+\caption[{{caption[1]}}]{% raw %}{{% endraw %}{{caption[0]}}{% raw %}}{% endraw %}
```

Comment:
Please keep in mind that the same behavior should be implemented for longtable (once the longtables are implemented through the styler).